{"name": "test_verify_wrong_login", "status": "passed", "attachments": [{"name": "log", "source": "cf382fd0-bdc6-4d98-823b-05dd872a07d5-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "f808c963-d267-42ca-af5d-20c92d87bc7f-attachment.txt", "type": "text/plain"}], "start": 1705398755088, "stop": 1705398758214, "uuid": "5a591e34-1ee3-41c3-89b1-bc79c91dd0e9", "historyId": "d2f9652812d69cd910ce2959677a0452", "testCaseId": "d2f9652812d69cd910ce2959677a0452", "fullName": "TestSuite.LoginTest#test_verify_wrong_login", "labels": [{"name": "tag", "value": "run(order=5)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "GESLLR0D3LS8"}, {"name": "thread", "value": "13732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}